.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ executors:
     environment:
       IMAGE_NAME: bukuserver/bukuserver
     docker:
-      - image: circleci/buildpack-deps:stretch
+      - image: cimg/base:current
```

Comment:
This image [had been deprecated](https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034):
![suggested replacement](https://github.com/user-attachments/assets/b4d7fa81-5028-458f-a277-90ad092fe4e3)